output_template/python/usb_camera_demo.py

```diff
@@ -79,6 +80,34 @@ def add_class_label(canvas,
     cv2.putText(canvas, text, dl_corner, font, font_scale, font_color, line_type)
 
 
+def label_to_color_image(results, length):
+    """Adds color defined by the dataset colormap to the label.
+
+    Args:
+        results: A 2D array with integer type, storing the segmentation label.
+        length: Single value with int type. The number of classes.
+
+    Returns:
+        result: A 2D array with floating type. The element of the array
+            is the color indexed by the corresponding element in the input label
+            to the PASCAL color map.
+
+    Raises:
+        ValueError: If label is not of rank 2 or its value is larger than color
+            map maximum entry.
+    """
+    if results.ndim != 4:
+        raise ValueError('Expect 4-D input results (1, height, width, classes).')
+
+    colormap = np.array(get_color_map(length), dtype=np.uint8)
```

Comment:
Why do you get colormap for each a image ? 🤔🤔🤔 What situation do you assume ?
In my viewpoint, to initialize colormap only once is enough on somewhere.

Reply:
Oh, my bad, I will fix them with commit b74bce1 for only initializing the colormap once.